### DISCUSSION TITLE
units of third of force constants

### DISCUSSION THREAD

**Discussion by rfhari** [ttadano/alamode#232] [Q&A] [answered]
Hi,

I'm working on calculating the second- and third-order force constants (FC2 and FC3) for AlN and GaN using ALAMODE with VASP. I need these force constants in ShengBTE format so I can use AlmaBTE to calculate the thermal conductivity of AlxGa(1-x)N. I understand that I can output FC2 in the `.hessian` file and rewrite it into ShengBTE format, and I can use the "FC3_SHENGBTE" tag to obtain FC3 in ShengBTE format as well.

However, I'm unsure about the units of FC3. Are the values in `PREFIX.FORCE_CONSTANT_3RD` given in Ryd/Bohr^3, or are they converted to eV/A^3? For FC2, this seems clearer as the header of the `.hessian` file specifies Ryd/Boh^2, but for FC3, there is no explicit mention of units in the `PREFIX.FORCE_CONSTANT_3RD` file. It would really great if you can clarify this! 

Thanks!

**Comment by ttadano** [comment, marked as answer]
The unit of FC3 saved in `PREFIX.FORCE_CONSTANT_3RD` is eV/Å^3.
